.build-tools/cmd/zz-e2e-perf.go

```diff
@@ -311,6 +311,19 @@ func (c *cmdE2EPerf) buildDockerImage(cachedImage string) error {
 		"-t", destImage,
 		filepath.Join(appDir, c.flags.Name, "."),
 	)
+
+	if c.flags.TargetArch == "arm64" {
```

Comment:
Should this be a switch instead?

```
args := []string{
		"build",
		"-f", dockerfile,
		"-t", destImage,
		filepath.Join(appDir, c.flags.Name, "."),
	}
	switch c.flags.TargetArch {
	case "arm64":
		args = append(args, "--platform", "linux/arm64/v8")
		// Add --load if needed>?
	case "amd64":
		args = append(args, "--platform", "linux/amd64")
	default:
		args = append(args, "--platform", "linux/amd64")
	}
	e := exec.Command("docker", args...)
```

This way should also be possible to build amd64 images on an arm64 machine!